CODE_OF_CONDUCT.md

```diff
@@ -1,66 +1,43 @@
-
-- maintain the confidentiality with regard to the reporter of an incident
-- to participate in the path for escalation as outlined in
-  the section on Escalation when required.
-
-#### Project Reporting Emails
-
-* AMP Project - `code-of-conduct@amp.dev`
-* Appium - [email maintainers]
-* Architect - 
-* Dojo - [reporting form](https://docs.google.com/forms/d/1ZiCUv3FsarWetoQ8J5QnCdIrHLj0xMmwNSLCyBkio7c/viewform?edit_requested=true)
-* Electron - `coc@electronjs.org`
-* ESLint - 
-* Esprima - 
-* Express.js - `express-coc@lists.openjsf.org`
-* Fastify - `hello@matteocollina.com` or `tommydelved@gmail.com`
-* Grunt - 
-* HospitalRun - `hello@hospitalrun.io`
-* Interledger.js - 
-* Intern - 
-* JerryScript - 
-* jQuery - 
-* libuv - 
-* LoDash - 
-* messageformat - 
-* Marko - 
-* Mocha - `report@lists.openjsf.org`
-* Moment.js - 
-* Node.js - `report@nodejs.org`
-* Node-RED - `team@nodered.org`
-* nvm - 
-* Pep - 
-* QUnit - 
-* Webdriver.io - [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
-* Webhint - `support@webhint.io`
-* webpack - [redirects here]
-
-### Foundation Spaces
-
-For reporting issues in spaces managed by the OpenJS Foundation, for example, repositories within the OpenJS organization, use the email `report@lists.openjsf.org`. The Cross Project Council (CPC) is responsible for managing these reports and commits to:
-
-- maintain the confidentiality with regard to the reporter of an incident
-- to participate in the path for escalation as outlined in
-  the section on Escalation when required.
-
-## Escalation
-
-The OpenJS Foundation maintains a Code of Conduct Panel (CoCP). This is a foundation-wide team established to manage escalation when a reporter believes that a report to a member project or the CPC has not been properly handled. In order to escalate to the CoCP send an email to `coc-escalation@lists.openjsf.org`.
+The OpenJS Foundation and its member projects use [Contributor Covenant v2.0](https://contributor-covenant.org/version/2/0/code_of_conduct) as their code of conduct. The full text is included [below](#contributor-covenant-code-of-conduct) in English, and [translations](https://www.contributor-covenant.org/translations) are available on the Contributor Covenant website.
 
+## Commitment
+
+All recipients of reports commit to:
+
+- maintain the confidentiality with regard to the reporter and victim of an incident, and
+- participate in the path for escalation as outlined in the section on Escalation when required.
+
+
+## Report an issue in a project
+
+1. To report an issue in one of the projects listed below, please use the method provided. The project itself is responsible for managing these reports.
+    * **AMP Project:** <code-of-conduct@amp.dev>
+    * **Appium:** email maintainers
```

Comment:
That needs to be fixed by at least providing a link

Reply:
Looking more closely, it seems Appium hasn't updated it CoC in while, so there's more to fix, here: https://github.com/appium/appium/blob/master/CONDUCT.md

Reply:
@tobie I was about to land this but not sure if it should wait on this?

Reply:
Going to land and it can be fixed up in a follow on PR.